### ISSUE TITLE
Check if import came from jest

### ISSUE THREAD

**Issue by waynevanson** [jest-community/eslint-plugin-jest#1081]
Related to #556

I'm writing a property based testing library and I want to test a function called `test`.
I import it and use it in a test, the eslint rules do not recognize that this is non jest import.

```ts
import { test } from "./test-fn"

describe(test, () =>{
  it("should do something good", () => {
    // two errors:
    //
    // Test is missing function arguments `eslint jest/no-disabled-tests`
    // Test has no assertions `eslint jest/expect-expect`
    expect(test({})).toBeDefined()
  })
})
```

This code base looks neat, great work!

Happy to contribute if this feature is ready to be worked on.

**Comment by G-Rath** [comment]
Yeah this is a duplicate of #556 - feel free to take a whack at it if you like (I've been meaning to but haven't had the time)
